### PR TITLE
Port MVVM Toolkit docs

### PR DESCRIPTION
**Pull request by Sergio0694**
This PR ports over the MVVM Toolkit docs, with latest updates.

### CODE REVIEW

**Comment by PRMerger14** [comment]
Microsoft authors of articles need to make updates through the private repository, so the updates can be staged and validated by current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged. 

It's easy to access the private repo version of your articles in GitHub. Just add *-pr* to the URL, as follows: 

 &nbsp;Public repo URL: `https://github.com/MicrosoftDocs/{docs-repo}` 
 &nbsp;Private repo URL: `https://github.com/MicrosoftDocs/{docs-repo-pr}` 

 We have closed this pull request. Please resubmit the updates through the private repo. For more information, see the [contributor guide](https://docs.microsoft.com/contribute/).

**Comment by michael-hawker** [comment]
Um @brminnick we removed the private repo didn't we? @davidbritch any idea what happened here?

**Comment by brminnick** [comment]
Haha - we sure did delete it! I’ve found that we can just re-open the PRs as a workaround for now.

**Comment by Sergio0694** [comment]
Ahah that works for me! 🤣
Thanks Brandon!

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 Changed Files | This PR contains more than 10 changed files.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Image File | This PR added or updated an image file(s).
 Index File | This PR contains a change to the index file.

**Comment by michael-hawker** [comment]
@Sergio0694 check the main index/toc files as well as I put jumps to the MVVM docs there as well too: https://github.com/MicrosoftDocs/CommunityToolkit/tree/main/docs

File: docs/dotnet/TOC.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ items:
 - name: MVVM Toolkit
   items:
     - name: Introduction
-      href: /windows/communitytoolkit/mvvm/introduction
+      href: /mvvm/introduction
```
Comment (michael-hawker) [inline docs/dotnet/TOC.yml:11]:
```suggestion
      href: mvvm/introduction
```

The initial slash roots it, so you need to remove it to do the relative path or do `/dotnet/communitytoolkit/dotnet/mvvm`

Also check the index.yml and toc.yml files in the next folder up as well. 😉

I forgot about creating the folder within the docs here and how that'll double-`dotnet` us in urls:

https://docs.microsoft.com/en-us/dotnet/communitytoolkit/dotnet

Did we want it that way? I think if we want to flatten we'd have a folder in the docs for each sub-package like `mvvm`, `high-performance`, and `diagnostics`. Though I don't think we could have a separate TOC that way 🙁

Thoughts? (We just have the one index page and its new, so we'll want to fix this now over later.

File: docs/dotnet/mvvm/RelayCommand.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    </StackPanel>
+</Page>
+```
+
+The `Button` binds to the `ICommand` in the viewmodel, which wraps the private `IncrementCounter` method. The `TextBlock` displays the value of the `Counter` property and is updated every time the property value changes.
+
+## Examples
+
+- Check out the [sample app](https://github.com/windows-toolkit/MVVM-Samples) (for multiple UI frameworks) to see the MVVM Toolkit in action.
+- You can also find more examples in the [unit tests](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/rel/7.1.0/UnitTests/UnitTests.Shared/Mvvm).
```
Comment (michael-hawker) [inline docs/dotnet/mvvm/RelayCommand.md:74]: Check for the old org name and repo links for this stuff, these should be updated (there may be others).

File: docs/dotnet/mvvm/Introduction.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+This package targets **.NET Standard** so it can be used on any app platform: UWP, WinForms, WPF, Xamarin, Uno, and more; and on any runtime: .NET Native, .NET Core, .NET Framework, or Mono. It runs on all of them. The API surface is identical in all cases, making it perfect for building shared libraries.
+
+Additionally, the MVVM Toolkit also has a **.NET 5** target, which is used to enable more internal optimizations when running in on the .NET 5 runtime. The public API surface is identical in both cases, so NuGet will always resolve the best possible version of the package without consumers having to worry about which APIs will be available on their platform.
+
+To install the package from within Visual Studio:
+
+1. In Solution Explorer, right-click on the project and select **Manage NuGet Packages**. Search for **Microsoft.Toolkit.Mvvm** and install it.
+
+    ![NuGet Packages](../images/get-started/manage-nuget-packages.png "Manage NuGet Packages Image")
```
Comment (michael-hawker) [inline docs/dotnet/mvvm/Introduction.md:28]: Maybe we should create an issue around having a centralized managing NuGet package page we can link to across all toolkit getting started docs?

File: docs/dotnet/index.md [hunk 4]
```diff
@@ -26,9 +26,9 @@ Feel free to browse the documentation using the table of contents on the left si
 [repo-url]: https://aka.ms/toolkit/dotnet ".NET Community Toolkit GitHub Repository"
 [dotnet-foundation]: https://www.dotnetfoundation.org/ ".NET Foundation Home Page"
 [windows-community-toolkit]: /windows/communitytoolkit/ "Windows Community Toolkit Documentation"
-[mvvm-toolkit-intro]: /windows/communitytoolkit/mvvm/introduction "MVVM Toolkit Introduction"
+[mvvm-toolkit-intro]: /mvvm/introduction "MVVM Toolkit Introduction"
```
Comment (michael-hawker) [inline docs/dotnet/index.md:29]: Same here, these are broken.

**Review by michael-hawker — CHANGES_REQUESTED** [review]

File: docs/dotnet/TOC.yml (hunk 1 above)
Comment (Sergio0694) [reply docs/dotnet/TOC.yml]: Ah, whoops! Fixed, thank you! 😄

File: docs/dotnet/mvvm/Introduction.md (hunk 3 above)
Comment (Sergio0694) [reply docs/dotnet/mvvm/Introduction.md:28]: Yup, that'd be great. No point in duplicating this 🙂

File: docs/dotnet/mvvm/RelayCommand.md (hunk 2 above)
Comment (Sergio0694) [reply docs/dotnet/mvvm/RelayCommand.md]: Updated all of them in b609b995fc90f4472fb142a2f5029855ff497245 👍

**Comment by michael-hawker** [comment]
@Sergio0694, I think the issue is if it's a file we need to add the `.md` at the end, think that's why the validator can't find them? (even though the links are working now)

File: docs/dotnet/TOC.yml (hunk 1 above)
Comment (michael-hawker) [reply docs/dotnet/TOC.yml]: Thoughts on the doc url duplication?

File: docs/dotnet/mvvm/Introduction.md (hunk 3 above)
Comment (michael-hawker) [reply docs/dotnet/mvvm/Introduction.md:28]: Opened #52

**Comment by michael-hawker** [comment]
Here:

https://github.com/MicrosoftDocs/CommunityToolkit/blob/46c0ae00a64bd595c548fe37f4b06d08fe69f9fb/docs/TOC.yml#L11

and here:

https://github.com/MicrosoftDocs/CommunityToolkit/blob/46c0ae00a64bd595c548fe37f4b06d08fe69f9fb/docs/index.yml#L13-L30

are the other links to update (not sure if you want to change the getting started link yet, but included the broader permalink here for illustration)

**Comment by PRMerger3** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 Changed Files | This PR contains more than 10 changed files.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Image File | This PR added or updated an image file(s).
 Index File | This PR contains a change to the index file.

**Comment by Sergio0694** [comment]
@michael-hawker weird, I fixed my edits and now it seems fine, even though there's still some absolute links without .md 🤷

**Comment by davidbritch** [comment]
@Sergio0694 @michael-hawker @brminnick 

> Um @brminnick we removed the private repo didn't we? @davidbritch any idea what happened here?

When there's a private repo, PRMerger is configured to make Microsoft FTEs use the private repo (the idea being that the public repo is only for community contrib.). But there isn't a private repo. So PRMerger has made a mistake. I'll have a look if anything can be done about that.

File: docs/dotnet/TOC.yml (hunk 1 above)
Comment (michael-hawker) [reply docs/dotnet/TOC.yml]: We discussed with @brminnick and decided to flatten the urls down. I've updated the PR for that so we'll test out how it works. I also opened issue #54 as I think we need to fix the breadcrumbs file somehow too, but I don't know how that works yet.

**Comment by PRMerger9** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Deleted File(s) | This PR deletes Markdown or YAML files owned by another author, or json file(s). The pull request must contain a comment from the pull request author confirming that all the file deletions are intentional before the pull request can be merged.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 Changed Files | This PR contains more than 10 changed files.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Image File | This PR added or updated an image file(s).
 Json File | This PR contains json file(s).

**Review by michael-hawker — APPROVED** [review]
Looks like we can't test the redirection stuff in the review site. So, we'll see how that works.

If we need general/overview docs for the toolkits as a whole, we can create a `general` directory or something when needed I think.

**Pull request merged**